packages/security/src/deviceTrust/checks/NonRootedCheck.ts

```diff
@@ -28,12 +30,12 @@ export class NonRootedCheck implements SecurityCheck {
       }
 
       document.addEventListener("deviceready", () => {
-        if (!IRoot) {
-          reject(new Error("Could not find plugin IRoot."));
+        if (!IRoot || !device) {
+          reject(new Error("Could not find plugin for Root Check"));
           return;
         }
-
-        IRoot.isRooted((rooted: number) => {
+        const isRootedCheck = device.platform === "Android" ? IRoot.isRootedRedBeer : IRoot.isRooted;
```

Comment:
Check `isCordovaAndroid` method in core that could be used here.

Reply:
Good catch :+1: